packages/datatrak-web/src/views/SurveySelectPage/SurveyCountrySelector.tsx

```diff
@@ -40,7 +40,7 @@ const CountrySelectWrapper = styled.div`
 
 interface SurveyCountrySelectorProps {
   countries: Entity[];
-  selectedCountry?: Entity | null;
+  selectedCountry?: Pick<Entity, 'code'> | null;
```

Comment:
Only `code` is needed here, and wanted a type that works both with `Country` and `Entity`